### PR TITLE
main: Add initial commands parsing and execution

### PR DESCRIPTION
**Pull request by 12345ieee**
Pretty easy solution for #246 .

I don't know if we need autotools magic to have `strtok_r()` available, or if you guys know of a suitable C-standard-only replacement.

### CODE REVIEW

**Comment by bkazemi** [comment]
No worries, `strtok_r()` is a POSIX-compliant function.

**Comment by 12345ieee** [comment]
That I know, I'm just not sure if I need to add some horrific macro in `configure.ac` to check for it.

**Comment by bkazemi** [comment]
All modern Unix OS's are POSIX-compliant (enough anyway), therefore there is no need to check.

File: main.c [hunk 1]
```diff
@@ -176,6 +183,32 @@ int main(int argc, char **argv)
         show_user("Please enter current value, or \"help\" for other commands.\n");
     }
 
+    /* execute commands passed by `-c`, if any */
+    char *line;
+    char *saveptr = NULL;
```
Comment (bkazemi) [inline main.c]: It'd probably be cleaner to declare these inside the for loop.

Comment (bkazemi) [inline main.c]: It'd probably be cleaner to declare these inside the for loop.

**Pull request merged**